hil/api.py

```diff
@@ -319,11 +320,11 @@ def node_delete(node):
     get_auth_backend().require_admin()
     node = get_or_404(model.Node, node)
     if node.project:
-        raise errors.BlockedError(
+        raise errors.Error(
```

Comment:
I assume this was supposed to be `AttachedResourceError`.

Reply:
I don't know how I skipped these two this but I've corrected them now. Will push in the future commits.